package.json

```diff
@@ -72,6 +72,7 @@
     "lerna-semantic-release": "^9.1.0",
     "lint-staged": "^7.0.4",
     "lodash": "^4.17.0",
+    "mutation-observer": "^1.0.3",
```

Comment:
this can be moved to `packages/console/package.json` dependencies now i believe.

Reply:
sorry - this is only used in tests. Nevermind ;)

Reply:
yes, jest tests are configured once per project